packages/modules/viewer-react/src/components/BaseViewer.tsx

```diff
@@ -107,6 +108,9 @@ export const BaseViewer: React.FC<ViewerProps> = ({
   // TODO not signed in message / loader
   return (
     <ErrorBoundary>
+      {(!authorized || !iModelJsInitialized) && (
```

Comment:
if authorized and imodeljs not initialized, we show "please sign in" which makes no sense.

![image](https://user-images.githubusercontent.com/11051042/127204652-2df2da48-9651-4897-a7bd-13c1d1e1b715.png)

Wonder if we should only initialize imjs after youre authorized, and we should show a spinner when initializing

Reply:
lets disregard delaying the initialization to after being authenticated, i dont like the coupling we have here with IModelApp.authorizationClient and imjsInitialization.